plugins/git/gitignore.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+# We use `mkVimPlugin` to avoid having a `settings` option.
```

Comment:
Wouldn't it be better to provide a way for `mkNeovimPlugin` to do this? E.g. a boolean parameter

Reply:
Maybe. I recently thought about merging those two helpers as they are very redundant.

Reply:
Not sure that's necessary for this PR.

As this plugin doesn't need us to call `setup()` it probably isn't an issue to use `mkVimPlugin` anyway.

Maybe just also mention that in the comment, to avoid confusion?

Reply:
I clarified the comment. Will think about unifying the helpers later :)